kedro/extras/datasets/pandas/sql_dataset.py

```diff
@@ -278,13 +285,19 @@ class SQLQueryDataSet(AbstractDataSet):
 
     """
 
-    def __init__(
-        self, sql: str, credentials: Dict[str, Any], load_args: Dict[str, Any] = None
+    def __init__(  # pylint: disable=too-many-arguments
+        self,
+        sql: str = None,
+        filepath: str = None,
+        credentials: Dict[str, Any] = None,
```

Comment:
I'm not sure `None` makes sense as a default argument here given the `if not (credentials and "con" in credentials and credentials["con"])` check... but I guess you needed to specify it given the `sql` and `filepath` arguments preceding it have default values?